src/main/scala/com/cloudera/sparkts/TimeSeriesStatisticalTests.scala

```diff
@@ -246,4 +246,34 @@ object TimeSeriesStatisticalTests {
     }
     diffsSum / residsSum
   }
+  
+  /**
+   * Breusch-Godfrey test for serial correlation in a model
+   *
+   * @return The Breusch-Godfrey statistic and p value. The statistic asymptotically 
+   * follows an X^2 distribution with maxLag degrees of freedom, and provides a test for the null hypothesis
```

Comment:
Keep lines under 100 characters.  Also, I'd move the bulk of this text up and only keep "The Breusch-Godfrey statistic and p value." after the `@return`